lib/internal/readline/interface.js

```diff
@@ -198,6 +203,8 @@ function InterfaceConstructor(input, output, completer, terminal) {
   this[kSubstringSearch] = null;
   this.output = output;
   this.input = input;
+  this.undoStack = [];
+  this.redoStack = [];
```

Comment:
I don't think those should be public.

Reply:
Uh, I'm not quite sure I fully understand what you mean. Shouldn't these two properties be similar to `this.history`? I noticed that all of the properties referred in class `Interface` were defined in function `InterfaceConstructor` as public, and there was no property defined in class `Interface`. Should we break this consistency, or move other public properties into class `Interface` also?

Reply:
That makes sense. Agree with you.